mail/urls.py

```diff
@@ -16,4 +16,9 @@
     url(r'^api/v0/mail/course/(?P<course_id>[\d]+)/$', CourseTeamMailView.as_view(), name='course_team_mail_api'),
     url(r'^api/v0/mail/learner/(?P<student_id>[\d]+)/$', LearnerMailView.as_view(), name='learner_mail_api'),
     url(r'^api/v0/mail/automatic_email/$', AutomaticEmailView.as_view(), name='automatic_email_api'),
+    url(
```

Comment:
hrm so taking that approach seems to break the usage of `reverse` that was working before. In one of the views tests I'm just doing:

```py
cls.url = reverse('automatic_email_api')
```

that's now throwing a `django.urls.exceptions.NoReverseMatch`